addons/product_expiry/models/production_lot.py

```diff
@@ -48,13 +48,14 @@ def _get_dates(self, product_id=None):
         return res
 
     # Assign dates according to products data
-    @api.model
-    def create(self, vals):
-        dates = self._get_dates(vals.get('product_id') or self.env.context.get('default_product_id'))
-        for d in dates:
-            if not vals.get(d):
-                vals[d] = dates[d]
-        return super(StockProductionLot, self).create(vals)
+    @api.model_create_multi
+    def create(self, vals_list):
+        for vals in vals_list:
+            dates = self._get_dates(vals.get('product_id') or self.env.context.get('default_product_id'))
+            for d in dates:
+                if not vals.get(d):
+                    vals[d] = dates[d]
+        return super(StockProductionLot, self).create(vals_list)
```

Comment:
super()

Reply:
@sle-odoo I don't understand this comment. Can you explain? Thanks

Reply:
super(Class, self) is a python2 syntax, In python3 super() do the exactly same thing. So you could write 
```
return super().create(vals_list)
```
since v13 only support python3

Reply:
@amoyaux Thanks